config/config.js

```diff
@@ -2,24 +2,15 @@ require('dotenv').config()
 
 module.exports = {
   development: {
```

Comment:
In development and testing environments, U can leave what was there, U need DATABASE_URL only in production

Reply:
Okay, great!